src/App.tsx

```diff
@@ -1,19 +1,72 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useCallback,
+  useContext, useEffect, useRef, useState,
+} from 'react';
 import { AuthContext } from './components/Auth/AuthContext';
+import { Erorr } from './components/Erorr/Erorr';
```

Comment:
```suggestion
import { Error } from './components/Error/Error';
```
Try to avoid typos

Reply:
Not fixed. If your have some problems fill free to ask in fe_chat